### PR TITLE
Units

### PR DESCRIPTION
**Pull request by raemadeline**
Still needs an introduction, but I'd like to gather feedback on how easy this is to understand and if it make sense.

@embooglement @lydiabarnes @pagael @joelshawaddepar

### CODE REVIEW

**Comment by embooglement** [comment]
LGTM

File: articles/2016-09-15-units.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+![pixel example](../images/pixel.png)
+*Image courtesy of [wikipedia.](https://en.wikipedia.org/wiki/Pixel)*
+
+Generally when you're declaring the length of something in CSS, it will be using pixels (or `px`).
+
+## Percentage
+
+Percentages are *technically* not a unit of measurement, but they are the second most commonly used method to declare the length (or width, or margin, or anything) so it should still be discussed.
+
+When an element has `width: 50%;`, its width will be 50% of its direct parent's. If its a top-level element (meaning it has no container other than `<body>`), its width will be 50% of the window.
```
Comment (LydiaBarnes) [inline articles/2016-09-15-units.md:18]: I'd add a quick tidbit that browsers calculate percentages into pixels. If you grab the width of an image in the dom through js for example, you'll get a super precise px amount.

**Review by LydiaBarnes — COMMENTED** [review]
LGTMBC

File: articles/2016-09-15-units.md (hunk 1 above)
Comment (raemadeline) [reply articles/2016-09-15-units.md:18]: Done

**Pull request merged**